denonavr/denonavr.py

```diff
@@ -1028,7 +1028,7 @@ def _get_receiver_sources(self):
             receiver_sources = {}
             # Source determination from XML
             favorites = root.find(".//FavoriteStation")
-            if favorites:
+            if len(favorites):
```

Comment:
imo this looks awkward , `If favorites is not None` is better 😃

Reply:
just update it